libgit/autogit_node_wrappers.go

```diff
@@ -36,7 +36,8 @@ import (
 // * `tlfNode` allows auto-creation of subdirectories representing
 //   valid repository checkouts of the corresponding TLF, e.g.
 //   `.kbfs_autogit/private/chris/dotfiles`.  It wraps child nodes in
-//   two ways, as both a `readonlyNode` and a `repoNode`.
+//   two ways, as both a `readonlyNode` and a `repoNode`.  It allows
+//   repo directories to be removed via `RemoveDir`.
```

Comment:
Might want to add that this only happens if you have write permissions to the folder.